perf.cmd

```diff
@@ -0,0 +1,10 @@
+@echo off
```

Comment:
Right, the file is mostly here to serve as an example of just what you need to do: restore, then run, the PerfHarness project. I'm happy to rename the file, or outright remove it if it's more confusing than helpful.